try {
    report();
    console.log('todo esta bien')
    
}catch{
    console.log('Hay un error')
}

// En try mandamos la logica y si algo 
//falla entrara al catch y nos dara un error o accion que querramos que suceda si pasa.